lang/rs/examples/fwd/src/main.rs

```diff
@@ -25,10 +25,10 @@ enum Mode {
     Drop,
     RxOnly,
```

Comment:
Strange that we do not have an alias for drop as rx-only or rx-only with alias of drop. Should we define alias here?

Reply:
For RxOnly and Drop, Rust clap module automatically enum shows it as drop and rx-only. It doesn't allow to pass "Drop" or "RxOnly" in command line.

Reply:
I do not understand, but if it is not allowed then I guess it is OK.

Reply:
Rust enum variant should be in camel case.  Rust clap module will change enum variant "RXOnly" to "rx-only" by default.